gulpfile.js

```diff
@@ -0,0 +1,38 @@
+const gulp = require('gulp');
+const browserSync = require('browser-sync').create();
+ const browserify = require('gulp-browserify');
```

Comment:
Hound is flagging this error because there's a space next to the `const` keyword on line 3